uiowa/uiowa
# [INRC] Grants block

Resolves https://github.com/uiowa/uiowa/issues/7844

# How to test

```
ddev blt frontend && ddev blt ds --site=inrc.law.uiowa.edu && ddev drush @lawinrc.local config:import --source ../config/sites/inrc.law.uiowa.edu --partial -y && ddev drush @lawinrc.local cim -y && ddev drush @lawinrc.local en devel_generate && ddev drush @lawinrc.local devel-generate:content 40 --bundles grant && ddev drush @lawinrc.local uli /node/add/page
```
1. Place block on the page as a webmaster
2. Confirm filters work
3. Check view settings
4. Resync and test with real content from https://inrc.law.uiowa.edu/iowa-grants-guide

```
ddev blt frontend && ddev blt ds --site=inrc.law.uiowa.edu && ddev drush @lawinrc.local config:import --source ../config/sites/inrc.law.uiowa.edu --partial -y && ddev drush @lawinrc.local cim -y && ddev drush @lawinrc.local uli /node/add/page
```
